src/lib/dataEntities/retryableData.ts

```diff
@@ -0,0 +1,84 @@
+import { Interface } from '@ethersproject/abi'
+import { BigNumber } from 'ethers'
+
+export interface RetryableData {
+  from: string
+  to: string
+  l2CallValue: BigNumber
+  deposit: BigNumber
+  maxSubmissionCost: BigNumber
+  excessFeeRefundAddress: string
+  callValueRefundAddress: string
+  gasLimit: BigNumber
+  maxFeePerGas: BigNumber
+  data: string
+}
+
+/**
+ * Tools for parsing retryable data from errors
+ */
+export class RetryableDataTools {
+  public static ErrorTriggeringParams = {
```

Comment:
feel (not strongly) like this should live in the tests, not the lib

Reply:
I want these here so that people can use them to trigger the retryable data - added a better comment to explain this.